### PR TITLE
Add 1LHAASO catalog

### PR DESCRIPTION
**Pull request by QRemy**
Add 1LHAASO catalog in FITS format and the scrip to create the file from a csv table.

### CODE REVIEW

File: catalogs/make_1LHAASO_fits.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun 15 17:15:06 2023
+
+@author: qremy
+"""
```
Comment (bkhelifi) [inline catalogs/make_1LHAASO_fits.py:7]: Add the used version of Gammapy, that fixes the versions of numpy, astropy and pandas

**Comment by bkhelifi** [comment]
Thanks @QRemy. I put here the exchanges we had on slack...

Maybe one suggestion, from the paper T. X. Chen et al., Astrophysical Journal Supplement Series, 260:5, 2022 May and reference within (https://outerspace.stsci.edu/display/MASTDOCS/Common+Metadata):
one could add the following metadata: DATE-BEG, DATE-END, MJD-BEG, MJD-END, INSTRUME, OBSERVAT, LICENCE, LICENURL, REFERENC

**Review by registerrier — APPROVED** [review]
Thanks @QRemy !

as suggested by @bkhelifi , a CREATOR keyword giving the `gammapy.version` used to create the file might be useful too.

File: catalogs/make_1LHAASO_fits.py (hunk 1 above)
Comment (QRemy) [reply catalogs/make_1LHAASO_fits.py:7]: added to header via the CREATOR keyword

**Review by bkhelifi — APPROVED** [review]
Thanks a lot @QRemy

**Pull request merged**